src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -303,6 +311,16 @@
             4,
             ConfigDef.Width.LONG,
             FIELDS_WHITELIST_DISPLAY
+        ).define(
+          DB_TIMEZONE_CONFIG,
+          ConfigDef.Type.STRING,
+          DB_TIMEZONE_DEFAULT,
+          ConfigDef.Importance.HIGH,
+          DB_TIMEZONE_CONFIG_DOC,
+          DATAMAPPING_GROUP,
+          5,
+          ConfigDef.Width.MEDIUM,
+          DB_TIMEZONE_CONFIG_DISPLAY
```

Comment:
As mentioned below, we should provide a `ConfigDef.Validator` method that ensures that all values are indeed valid time zone names. This will help when configuring connectors, since an error message like `'{}' is not a valid name of a time zone` would appear in the context of `db.timezone`.